diff -urN xfce-mcs-plugins-4.4.0/configure.in xfce-mcs-plugins-4.4.0.new/configure.in
--- xfce-mcs-plugins-4.4.0/configure.in	2007-01-21 19:14:21.000000000 +0100
+++ xfce-mcs-plugins-4.4.0.new/configure.in	2007-01-21 19:14:52.000000000 +0100
@@ -76,7 +76,7 @@
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl pa pl pt_BR pt_PT ro ru sk sv ta tr uk vi zh_CN zh_TW])
+XDT_I18N([ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl pa pl pt_BR pt ro ru sk sv ta tr uk vi zh_CN zh_TW])
 
 dnl *********************
 dnl *** Check for X11 ***
--- xfce-mcs-plugins-4.4.1/po/LINGUAS.orig	2007-04-11 20:01:03.000000000 +0200
+++ xfce-mcs-plugins-4.4.1/po/LINGUAS	2007-04-11 20:01:17.000000000 +0200
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl pa pl pt_BR pt_PT ro ru sk sv ta tr uk vi zh_CN zh_TW
+ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl pa pl pt_BR pt ro ru sk sv ta tr uk vi zh_CN zh_TW
