--- xfce-mcs-plugins-4.0.5/configure.ac.orig	Sun Mar 14 22:40:00 2004
+++ xfce-mcs-plugins-4.0.5/configure.ac	Thu Jul  8 12:25:14 2004
@@ -26,7 +26,7 @@
 
 dnl Check for i18n support
-BM_I18N([xfce-mcs-plugins], [ar az bg ca cs de es es_MX fa_IR fi fr hi hu
-	 it ja ko lt ms nl no pl pt_BR pt_PT ru sv ta tr uk vi zh_CN zh_TW])
+BM_I18N([xfce-mcs-plugins], [ar az bg ca cs de es es_MX fa fi fr hi hu
+	 it ja ko lt ms nl nb pl pt_BR pt ru sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11
 BM_LIBX11_REQUIRE
