--- xfce-mcs-plugins-4.1.1/configure.ac.orig	2004-07-30 03:52:31.722206840 +0200
+++ xfce-mcs-plugins-4.1.1/configure.ac	2004-07-30 03:52:48.749618280 +0200
@@ -25,8 +25,8 @@
 AC_CHECK_FUNCS(opendir)
 
 dnl Check for i18n support
-BM_I18N([xfce-mcs-plugins], [ar az bg ca cs de es es_MX es fa_IR fi fr hi hu it
-	 ja ko lt ms nl pl pt_BR pt_PT ru sv ta tr uk vi zh_CN zh_TW])
+BM_I18N([xfce-mcs-plugins], [ar az bg ca cs de es es_MX es fa fi fr hi hu it
+	 ja ko lt ms nl pl pt_BR pt ru sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11
 BM_LIBX11_REQUIRE
